Detectors/ITSMFT/ITS/macros/test/CreateDictionaries.C

```diff
@@ -32,12 +32,12 @@
 #include <unordered_map>
 #endif
 
-void CheckTopologies(std::string clusfile = "o2clus_its.root",
-                     std::string hitfile = "o2sim_HitsITS.root",
-                     std::string collContextfile = "collisioncontext.root",
-                     std::string inputGeom = "",
-                     float checkOutliers = 2., // reject outliers (MC dX or dZ exceeds row/col span by a factor above the threshold)
-                     float minPtMC = 0.01)     // account only MC hits with pT above threshold
+void CreateDictionaries(std::string clusfile = "o2clus_its.root",
```

Comment:
could you add optional extraction of dX,dZ, and topology.hash() as discussed privately?